### PR TITLE
LOOC muting again

### PR DESCRIPTION
**Pull request by AnywayFarus**
## About The Pull Request

Back possibility to admin for LOOC mute.
For round, and like a ban.

## How This Contributes To The Skyrat Roleplay Experience

no way, I guess. Maybe less metagaming.

## Changelog

:cl: Vishenka0704
add: Added LOOC ban.
fix: fix LOOC mute(localy, on round length)
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Finally.
Admin tools to counter less-than-reliable people.
Woooooooooo.

File: code/modules/admin/verbs/admingame.dm [hunk 1]
```diff
@@ -84,7 +84,10 @@
 		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_OOC]'><font color='[(muted & MUTE_OOC)?"red":"blue"]'>OOC</font></a> | "
 		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_PRAY]'><font color='[(muted & MUTE_PRAY)?"red":"blue"]'>PRAY</font></a> | "
 		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_ADMINHELP]'><font color='[(muted & MUTE_ADMINHELP)?"red":"blue"]'>ADMINHELP</font></a> | "
-		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_DEADCHAT]'><font color='[(muted & MUTE_DEADCHAT)?"red":"blue"]'>DEADCHAT</font></a>\]"
+		//Skyrat Addition Begin - LOOC muting again.
+		body += "<A href='?_src_=holder;[HrefToken()];mute=[M.ckey];mute_type=[MUTE_DEADCHAT]'><font color='[(muted & MUTE_DEADCHAT)?"red":"blue"]'>DEADCHAT</font></a> | "
```
Comment (AnywayFarus) [inline code/modules/admin/verbs/admingame.dm:88]: changed \] to " | "

**Comment by weesnaw-ops** [comment]
i feel like LOOC is too vital of a feature to allow a ban of it. you can play without ooc. you can play without deadchat. but you can't tell people to not sexually harass you or shit without looc. this is a terrible terrible idea. just ban the players who misuse looc like youd ban anyone else

**Comment by AnywayFarus** [comment]
@weesnaw-ops 
well, only use the temporary option if you're all that way with it.. difficult. at least it would be useful for us as a fork, and this does not harm the build itself at all.

**Review by GoldenAlpharex — APPROVED** [review]
I don't plan on ever letting LOOC bans be handed out on Skyrat, I'm mostly getting this to reintroduce the lost ability to have LOOC mutes for people that are, say, complaining in LOOC about an ongoing ticket with an admin, which has happened many times in the past.

**Pull request merged**